src/technologies/j.json

```
@@ -17,6 +17,20 @@
     ],
     "website": "https://www.j2store.org"
   },
+  "Jadu": {
+    "cats": [
+      1
+    ],
+    "dom": "footer a[href*='jadu.net']",
```

```suggestion
```
Could we skip this?
It can give false positive results if technology is mentioned but not actually present.

---

```suggestion
    "headers": {"node": "^jadu-web-."},
```

This tech returns quite a unique header, so I would probably rely on this one.